packages/extras/src/lib/audio/AudioListener/AudioListener.svelte.d.ts

```diff
@@ -0,0 +1,13 @@
+import type { Props } from '@threlte/core'
+import type { SvelteComponentTyped } from 'svelte'
+import type { AudioListener as ThreeAudioListener } from 'three'
+
+export type AudioListenerProps = Props<ThreeAudioListener> & {
+  inViewport?: boolean
```

Comment:
viewport awareness will be removed in v6 and inserted back via a plugin.

Reply:
Removed viewport-related props and events